x/sync/client_test.go

```diff
@@ -12,6 +12,8 @@ import (
 
 	"github.com/golang/mock/gomock"
```

Comment:
nit: make import alphabetically ordered

Reply:
For avalanchego we order our imports as follows:
* standard library imports
* non-standard-library imports (separated by spaces)
* imports from avalanchego

I've adhered to that here, but I re-ordered the non-standard-library imports to be alphabetical